src/tsickle.ts

```diff
@@ -2021,6 +2021,13 @@ export function emitWithTsickle(
   };
 }
 
+/** Compares two strings and returns a number suitable for use in sort(). */
+function stringCompare(a: string, b: string): number {
```

Comment:
our locale should always be C, so `a.localeCompare(b)` should do it here, right? But this is probably safer.

Reply:
I considered using that, and kinda puzzled over whether locales could matter, and eventually decided just to not worry about it.  You might reasonably wonder why there is no ordinary string compare function, sigh.